src/Renderer/RendererOpenGL.cpp

```diff
@@ -764,6 +764,13 @@ std::vector<NAS2D::DisplayDesc> NAS2D::RendererOpenGL::getDisplayModes() const
 	return result;
 }
 
+NAS2D::DisplayDesc NAS2D::RendererOpenGL::getWindowDisplayDesc() const
```

Comment:
This could maybe use a bit of documentation. A screen has display mode settings. A window itself doesn't. It's maybe a little confusing to call this `getWindowDisplayDesc`. To confuse matters further, a window could be split across two different screens, and those screens might have different resolution settings. A bit more clarity on what this represents would be appreciated. Are these properties of the window, or of the display the window is on, and if split across multiple displays, which one?

Reply:
Properties of the *majority* of the window the screen is on...at least that's how it works in DirectX. SDL documentation is lacking when the window is across several monitors.

Will add documentation.